src/cr/cube/enums.py

```diff
@@ -143,17 +147,28 @@ class CUBE_MEASURE(enum.Enum):
     STDDEV = "stddev"
     SUM = "sum"
     VALID_OVERLAP = "valid_overlap"
-    VALID_COUNT_UNWEIGHTED = "valid_count_unweighted"
-    VALID_COUNT_WEIGHTED = "valid_count_weighted"
+    UNWEIGHTED_VALID_COUNT = "valid_count_unweighted"
+    WEIGHTED_VALID_COUNT = "valid_count_weighted"
 
 
-NUMERIC_MEASURES = frozenset(
+NUMERIC_CUBE_MEASURES = frozenset(
     (
-        CUBE_MEASURE.SUM,
         CUBE_MEASURE.MEAN,
+        CUBE_MEASURE.SUM,
         CUBE_MEASURE.STDDEV,
-        CUBE_MEASURE.VALID_COUNT_UNWEIGHTED,
-        CUBE_MEASURE.VALID_COUNT_WEIGHTED,
+        CUBE_MEASURE.UNWEIGHTED_VALID_COUNT,
+        CUBE_MEASURE.WEIGHTED_VALID_COUNT,
+    )
+)
+
+SECOND_ORDER_NUMERIC_MEASURES = frozenset(
```

Comment:
I think this could just be `NUMERIC_MEASURES` for conciseness. It would be unlikely to be confused with `NUMERIC_CUBE_MEASURES` I expect and we don't generally make a big deal about specifying "second-order" elsewhere. When we say "measure" we always mean one of these and when we mean "cube-measure" we always spell it out.

Plus, if this is imported by a client like exporter, they have no need for the concept of "second-order" measure since cube-measures are a `cr.cube` internal concept.